src/twisted/python/_release.py

```diff
@@ -39,14 +40,66 @@
 
 
 def runCommand(args, **kwargs):
-    """Execute a vector of arguments.
+    """
+    Execute a vector of arguments.
 
     This is a wrapper around L{subprocess.check_output}, so it takes
     the same arguments as L{subprocess.Popen} with one difference: all
     arguments after the vector must be keyword arguments.
+
+    @param args: arguments passed to L{subprocess.check_output}
+    @param kwargs: keyword arguments passed to L{subprocess.check_output}
+    @return: command output
+    @rtype: L{str}
     """
     kwargs['stderr'] = STDOUT
-    return check_output(args, **kwargs)
+    output = check_output(args, **kwargs)
+
+    # The check_output() function always returns bytes
+    # on Python 2 and 3, but we want this function to always return
+    # str.
+    if not isinstance(output, str):
+        output = output.decode("utf-8")
+    return output
+
+
+
+class FilePathStrContent(FilePath):
+    """
+    Wrapper around L{FilePath} to make dealing with
+    L{bytes} and L{str}
```

Comment:
`str` isn't really a distinct type anymore.  Do you mean `unicode`?  In general, this class seems like a bad idea.  It encourages lack of clarity about the string type being operated on.

Put another way, this is really deferring the actual work of porting.